### ISSUE TITLE
output supporting reads

### ISSUE THREAD

**Issue by ywzhang071394** [PacificBiosciences/trgt-denovo#7]
Hi

I am just wondering if there is a way to output the supporting reads.

thank you!

**Comment by tmokveld** [comment]
Hi! Are you referring to the reads identified as having '_de novo_ evidence'? In the context of a trio, these would be the reads in the child that differ from the parental data. 

I can see how this could be useful. Would a log file formatted in this way help with your use case? Each tandem repeat ID would have an associated list of read IDs corresponding to the child BAM file.
```
>TRID_0
m64012_190920_173625/68354795/ccs
m64012_190920_173625/77923358/ccs
m64012_190921_234837/69010091/ccs
m64012_190920_173625/83036707/ccs
m64015_190922_010918/89917077/ccs
m64015_190922_010918/132056485/ccs
m64012_190921_234837/153750211/ccs
m64012_190920_173625/141820432/ccs
>TRID_1
...
```

**Comment by ywzhang071394** [comment]
yes, this log file would be very helpful. we are searching for repeat expansion in tumor samples and want to know whether the reads identified as having 'de novo evidence' come from the same allele. That's why we need a list of "supporting reads".

**Comment by ywzhang071394** [comment]
also, I noticed you are working on the analysis example of DUO mode. I am wondering if you have good parameters/cutoffs to determine the "de novo" repeat expansions in DUO mode.

**Comment by tmokveld** [comment]
The duo mode is indeed still a work in progress. At this stage, I don’t have finalized parameters or cutoffs for identifying _de novo_ repeat expansions in that context, but this is a area I aim to develop and document by early 2025. 

If you’d like, I can keep you updated as this develops, or feel free to share any specific insights or challenges you’ve encountered—your input is very welcome!

**Comment by ywzhang071394** [comment]
I am grateful that I can be kept updated. Thank you so much!

**Comment by ywzhang071394** [comment]
Apologies for reopening this issue, but I noticed that the clustering algorithm is unable to accurately measure the motif count, as demonstrated below. In the figure, there are clearly two reads supporting this repeat expansion event, yet the motif counts for both "a" and "b" show no difference in the table.


![image](https://github.com/user-attachments/assets/07448f02-73fa-4cce-961f-84df734c77a6)

`trid	genotype	denovo_coverage	allele_coverage	allele_ratio	a_coverage	a_ratio	mean_diff_b	denovo_status	per_allele_reads_a	per_allele_reads_b	a_dropout	b_dropout	index	a_MC	b_MC	a_AL	b_AL	b_overlap_coverage
chr4_72504833_72504951_trsolve	2	2	6	0.333	34	0.059	4.0	Y:?	28,6	17,17	N	N	1	31_0,33_0	31_0,33_0	120,127	120,128	16,16
`

**Comment by tmokveld** [comment]
This has been implemented in [version 0.2.2](https://github.com/PacificBiosciences/trgt-denovo/releases/tag/v0.2.2).

**Comment by ywzhang071394** [comment]
Thank you for the update!
